lib/challenge.js

```diff
@@ -19,7 +19,18 @@
   // - park
 
 // Answer Starts Here
-
+// 'Calendar'
+// Description : Our app manages events on a calendar.
+// Every 'calendar' has
+// - a Year
+// - a Month (each Year has these)
+// - a Week (each Month has these)
+// - a Day (each Week has these)
```

Comment:
Calendar is a reasonable abstraction - can you name two others?

Or are Year, Month, Week, and Day your other abstractions?